model_zoo/TCIA_PROSTATEx_Prostate_MRI_Anatomy_Model.ipynb

```diff
@@ -692,6 +692,7 @@
     "compare_image = itk.GetImageFromArray(compare_model_expert.astype(np.float32))\n",
     "compare_image.CopyInformation(seg_image_prep)\n",
     "\n",
+    "# Comment this line as a workaround for https://github.com/Project-MONAI/tutorials/issues/1694\n",
```

Comment:
I think no need this comment anymore?

Thanks.

Reply:
Thanks Nic, fixed in the latest commit.